admin_guide/managing_networking.adoc

```diff
@@ -670,24 +666,55 @@ redirecting rules.
 proxy. Note that this is specified as an environment variable in the
 HTTP proxy container, not with the other environment variables in the
 init container.
-====
++
+You can specify any of the following for the `EGRESS_HTTP_PROXY_DESTINATION`
+value. You can also use `*`, meaning "allow connections to all remote
+destinations". Each line in the configuration specifies one group of connections
+to allow or deny:
++
+- An IP address (eg, `192.168.1.1`) allows connections to that IP address.
+- A CIDR range (eg, `192.168.1.0/24`) allows connections to that CIDR range.
+- A host name (eg, `www.example.com`) allows proxying to that host.
+- A domain name preceded by `\*.` (eg, `*.example.com`) allows proxying to that domain and all of its subdomains.
+- A `!` followed by any of the above denies connections rather than allowing them
+- If the last line is `*`, then anything that hasn't been denied will be allowed. Otherwise, anything that hasn't been allowed will be denied.
 
-You can specify a variety of things for
-`EGRESS_HTTP_PROXY_DESTINATION`. The simplest is to just specify `*`,
-meaning "allow connections to all remote destinations". Other than
-that, each line specifies one group of connections to allow or deny:
+. Ensure other pods can find the pod's IP address by creating a service to point
+to the egress router:
++
+[source, yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  name: egress-1
+spec:
+  ports:
+  - name: http
+    port: 8080 <1>
+  - name: https
+    port: 443
+  type: ClusterIP
+  selector:
+    name: egress-1
+----
+<1> Ensure the `http` port is always set to `8080`.
 
-- An IP address (eg, `192.168.1.1`) will allow connections to that IP address.
-- A CIDR range (eg, `192.168.1.0/24`) will allow connections to that CIDR range.
-- A hostname (eg, `www.example.com`) will allow proxying to that host.
-- A domain name preceded by `\*.` (eg, `*.example.com`) will allow proxying to that domain and all of its subdomains.
-- A `!` followed by any of the above will deny connections rather than allowing them
-- If the last line is `*`, then anything that hasn't been denied will be allowed. Otherwise, anything that hasn't been allowed will be denied.
+. In the pod configuration, set the service as a variable:
```

Comment:
The `http_proxy`/`https_proxy` environment variable is a very common convention, but there's nothing that *requires* apps to use it. In the case of apps/commands that don't use those variables you would probably either need to pass a command-line option or set some option in a config file.

You can probably get away with "in most cases" rather than "in many cases". All you can do in the other cases is refer them to the documentation for whatever tool it is that they're using in the pod.

Maybe start out just saying to use the environment variables, and then afterward have a note about "if this doesn't work then consult the documentation for the software you are running in the pod to see how to configure it to use an HTTP proxy".